extensions/amp-story/1.0/amp-story-request-service.js

```diff
@@ -30,7 +30,7 @@ export class AmpStoryRequestService {
     this.xhr_ = Services.xhrFor(win);
 
     /** @const @type {function():(!Promise<!JsonObject>|!Promise<null>)} */
-    this.loadShareConfig = once(() => this.loadShareConfigImpl_());
+    this.loadShareConfig = once(() => this.loadConfigImpl());
```

Comment:
Looks like you've generalized the methods in this file to be used for also loading non-share configs, but there are still a lot of references to "share" throughout the file's comments & variable names that imply otherwise. Could you update them?

Reply:
There is actually a Share config method that is specifically used for loading a shared config (not what we are doing in this particular PR). Hence why I chose to change only the method name of the method we are using to load our config, which is used by the share config loader, but also used by the shopping config loader.